src/pages/edge/adobe-journey-optimizer-decisioning/optimize-configuration-view.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Optimize Setup Validation view
+description: An overview of how the Optimize datastream and sdk setup can be validated.
+keywords:
+- Datastream Validation
+- SDK Validation
+---
+
+# Optimize Setup Validation view
+
+## Overview
+
+You can use the Optimize Setup Validation view to check if your application has been set up successfully to use Adobe Experience Platform Optimize. The view validates all pre-requisites have been met and offers feedback on how to fix your setup in cases where requirements have been missed.
```

Comment:
`Optimize Setup Validation view` ->`Decisioning validation view`
`your application has been set up successfully to use Adobe Experience Platform Optimize.` -> `your mobile application has been set up successfully to use Adobe Experience Platform Optimize.`